diff --git a/plugins/win-dshow/virtualcam-module/virtualcam-module.cpp b/plugins/win-dshow/virtualcam-module/virtualcam-module.cpp
index 23e0ee4..67b29c0 100644
--- a/plugins/win-dshow/virtualcam-module/virtualcam-module.cpp
+++ b/plugins/win-dshow/virtualcam-module/virtualcam-module.cpp
@@ -159,7 +159,7 @@ static bool RegServers(bool reg)
 	}
 
 	if (reg) {
-		return RegServer(CLSID_OBS_VirtualVideo, L"OBS Virtual Camera", file);
+		return RegServer(CLSID_OBS_VirtualVideo, L"Producer Virtual Camera", file);
 	} else {
 		return UnregServer(CLSID_OBS_VirtualVideo);
 	}
@@ -183,7 +183,7 @@ static bool RegFilters(bool reg)
 		rf2.cPins = 1;
 		rf2.rgPins = &AMSPinVideo;
 
-		hr = fm->RegisterFilter(CLSID_OBS_VirtualVideo, L"OBS Virtual Camera", &moniker,
+		hr = fm->RegisterFilter(CLSID_OBS_VirtualVideo, L"Producer Virtual Camera", &moniker,
 					&CLSID_VideoInputDeviceCategory, nullptr, &rf2);
 		if (FAILED(hr)) {
 			return false;
